crates/subspace-networking/src/behavior/persistent_parameters.rs

```diff
@@ -70,40 +70,27 @@ impl Clone for Box<dyn NetworkingParametersRegistry> {
     }
 }
 
-/// Networking manager implementation with bootstrapped addresses. All other operations muted.
+/// Networking manager implementation with NOOP implementation.
```

Comment:
Now I'm wondering if from API perspective it would be better to use `Option<Box<T>>` and this implementation would be private and only used internally during instantiation, outside users would be able to use `.unwrap_or_default()` rather than `.unwrap_or(Ok(StubNetworkingParametersManager.boxed()))`. WDYT?

Reply:
Yes, that is exactly what I meant. Just make UX a bit better.